config/config.yaml

```diff
@@ -11,6 +11,10 @@ kraken:
     litmus_uninstall: False                                # If you want to uninstall litmus if failure
     litmus_uninstall_before_run: True                      # If you want to uninstall litmus before a new run starts
     chaos_scenarios:                                       # List of policies/chaos scenarios to load
+        -   arcaflow_scenarios:
```

Comment:
@tsebastiani thoughts on removing https://github.com/redhat-chaos/krkn/blob/main/config/config_arcaflow.yaml to have everything in one place?

Reply:
Thought was already removed. :+1: